Conflict data resolution enhancement during data import (#16720)
pingcap/docs: tidb-lightning/tidb-lightning-logical-import-mode-usage.md

| :-- | :-- | :-- |
| `"replace"` | Replacing existing data with new data. | `REPLACE INTO ...` |
| `"ignore"` | Keeping existing data and ignoring new data. | `INSERT IGNORE INTO ...` |

> | `"error"` | Pausing the import when conflicting data is detected. | `INSERT INTO ...` |
> |  `""`  | Be converted to `"error"`, which means pausing the import when conflicting data is detected.  |  None   |

Author: ```suggestion
| `"error"` | Terminating the import when conflicting data is detected. | `INSERT INTO ...` |
|  `""`  | Be converted to `"error"`, which means terminating the import when conflicting data is detected.  |  None   |
```

| `"replace"` | Replacing existing data with new data. | `REPLACE INTO ...` |
| `"ignore"` | Keeping existing data and ignoring new data. | `INSERT IGNORE INTO ...` |

> | `"error"` | Pausing the import and reporting an error. | `INSERT INTO ...` |

Author: ```suggestion
| `"error"` | Pausing the import when conflicting data is detected. | `INSERT INTO ...` |
```

> |  `""`  | Be converted to `"error"`. Pausing the import and reporting an error.  |  None   |

Author: ```suggestion
|  `""`  | Be converted to `"error"`, which means pausing the import when conflicting data is detected.  |  None   |
```
